models/asymkey/ssh_key_authorized_keys.go

```diff
@@ -152,6 +152,10 @@ func RegeneratePublicKeys(ctx context.Context, t io.StringWriter) error {
 				return err
 			}
 		}
+		err = scanner.Err()
+		if err != nil {
+			return fmt.Errorf("scan: %w", err)
```

Comment:
If it returns here, the `f.Close` call is missing? Maybe resource leaking.

(And there seem to be more similar cases below)

Reply:
I agree, but this PR was merged.

Reply:
->  Fix some pending problems #29985